.rubocop.yml

```diff
@@ -16,7 +16,8 @@ Metrics/ClassLength:
 Metrics/BlockLength:
   ExcludedMethods: ['describe']
   Max: 30
-
+Metrics/CyclomaticComplexity:
+  Max: 10
```

Comment:
As it is written in the requirements, we SHOULD NOT change the linter file. We can conclude that It is a kind of optional requirement in the project. It is not must.

Reply:
As @kitpao said the linter files are not to be changed. 
Linters are a way to standardize coding practices within a company in a real-life scenario. To train you for that, Microverse has its own set of rules that you must comply with. 
Please change the method affected by this rule (refactor it)  instead of changing the linter file.
If you still have questions about this, feel free to send a message to student success.